effectsize_example.Rmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "Effect size"
+output: html_notebook
+---
+```{r}
+library(tidyverse)
+```
+
+
+## Data
+Assuming between-subjects design:
+```{r}
+set.seed(12)
+n <- 20
+data <- tibble(
+  Group = rep(c("A", "B"), each = n),
+  TaskCompletionTime_Sec = c(
+    rnorm(n, mean = 12, sd = 1),
+    rnorm(n, mean = 10, sd = 2)
+  )
+)
+```
+
+## Visualizing data
+
+```{r, fig.height=2, fig.width=3}
+pd <- position_dodge(0.2)
+data %>% 
+  ggplot(aes(x = Group, y = TaskCompletionTime_Sec)) +
+  geom_point(alpha = 0.2, position = pd) +
+  stat_summary(fun.data = "mean_cl_normal", geom = "pointrange") +
+  coord_flip() +
+  ylab("Task Completion Time (s)")
+
+
+```
+
+## Simple effect size
+
+```{r}
+data_A <- (data %>% filter(Group == "A"))[[2]]
```

Comment:
Using the numeric index here for a column is opaque and a little brittle to changes in the data frame